core/controller/src/main/scala/org/apache/openwhisk/core/entitlement/Entitlement.scala

```diff
@@ -295,6 +310,12 @@ protected[core] abstract class EntitlementProvider(
             if (failedResources.isEmpty) Future.successful(())
             else Future.failed(unauthorizedOn(failedResources.map(_._1)))
           })
+          .recoverWith {
+            case e =>
+              logging
+                .info(this, s"failed to check resources: ${e.getMessage}")
+              Future.failed(checkAuthorizationFailureOn(resources))
```

Comment:
What do we currently respond with if an inner authorization call fails? Just a generic error?